assets/js/components/notifications/BannerNotification/index.js

```diff
@@ -435,6 +444,10 @@ BannerNotification.propTypes = {
 	learnMoreURL: PropTypes.string,
 	learnMoreDescription: PropTypes.string,
 	learnMoreLabel: PropTypes.string,
+	learnMoreTarget: PropTypes.oneOf( [
```

Comment:
I considered using `PropTypes.oneOf( Object.values(LEARN_MORE_TARGET) )` here, but figured it's a bit clearer in terms of documenting the interface to list the values individually. Happy to change it though.